src/app/components/task-list/task-list.component.ts

```diff
@@ -17,16 +17,15 @@ import { RemoveButtonComponent } from 'src/app/remove-button/remove-button.compo
 export class TaskListComponent implements OnInit, OnDestroy {
   completedTasksSubscription = Subscription.EMPTY;
   tasksCompleted: number = 0;
+  deleteSubscription = Subscription.EMPTY;
   tasksSubscription = Subscription.EMPTY;
   tasks: Task[] = [];
```

Comment:
so here we would have an observable instead of just Task[], Observable<Task[]> (referring to comments below about using behaviorSubject and Observables or just the behaviorSubject